changelogs/unreleased/fix-manuf-order-plannification-methode-not-launched.yml

```diff
@@ -0,0 +1,3 @@
+---
+title: "Manuf order plannification methode not launched"
```

Comment:
Title not explicit enough. It can be for example "Manuf Order: Fixed a bug some plannifications processes were not launched" (Processes can even be detailed I think, but should be ok anyway)